blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -212,3 +228,74 @@ def test_join_type_promotion(sqla, sqlb):
     result = set(map(tuple, compute(expr, {t: sqla, s: sqlb}).execute().fetchall()))
     expected = set([(1, 'a', 'a'), (1, None, 'a')])
     assert result == expected
+
+
+def test_dist(nyc):
```

Comment:
kind of, they are tests that are using real data as opposed to hand-constructed data